### DISCUSSION TITLE
Skip "unloaded" tabs when using `toolkit.tabbox.switchByScrolling` (switch tabs via scrollwheel)

### DISCUSSION THREAD

**Discussion by tyler-dot-earth** [zen-browser/desktop#3372] [Ideas]
One feature from Sidebery that I would appreciate having is the ability to skip "unloaded" tabs when using `toolkit.tabbox.switchByScrolling` (switch tabs via scrollwheel).

Maybe this can be built as a Zen mod, though I haven't looked into it much.

For reference, here's the various tab scrolling options in Sidebery:

![image](https://github.com/user-attachments/assets/bf7574e2-3cd0-4ab7-a01b-fb849e314cb9)

**Comment by tunalad** [comment]
+1

scrolling over an unloaded tab by accident is pretty annoying
